amd64/root/start.sh

```diff
@@ -29,7 +34,11 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   fi
   
   # Set VNC password
-  echo "$DECONZ_VNC_PASSWORD" | tigervncpasswd -f > /root/.vnc/passwd
+  if [ "$DECONZ_VNC_PASSWORD_FILE" != 0 ]; then
+      cat "$DECONZ_VNC_PASSWORD_FILE" | tigervncpasswd -f > /root/.vnc/passwd
```

Comment:
Documentation for `DECONZ_VNC_PASSWORD_FILE` needs to be added to the README.

Reply:
It also needs to be stated how to get this file accessible from inside the docker ...

Reply:
Will be resolved in the next commit